.github/workflows/generate-cve.yml

```diff
@@ -56,9 +63,7 @@ jobs:
         run: |
           cd fleet
           go mod download
-          export VULNCHECK_API_KEY=${{ secrets.VULNCHECK_API_KEY }}
           go run cmd/cve/generate.go --db_dir ./cvefeed --debug
-          rm ./cvefeed/vulncheck.zip
```

Comment:
Why wasn't it failing (exit code 1 for `rm`)?

Reply:
good q, it was failing in my personal repo....taking a look

Reply:
a rebase cleaned that up